metricbeat/docs/running-on-docker.asciidoc

```diff
@@ -1,45 +1,25 @@
-[[running-in-container]]
-== Running Metricbeat in a Container
-
-ifeval::["{release-state}"=="released"]
-
-[NOTE]
-==================================================
-The https://github.com/elastic/beats-docker[official Docker images] for Beats
-are available from the Elastic Docker registry. To retrieve the images, simply
-issue the `docker pull` command:
-
-+docker pull docker.elastic.co/beats/metricbeat:{stack-version}+.
-
-The images are currently under development and should be considered
-alpha-quality. We strongly recommend that you do not run these images
-in a production environment.
-
-==================================================
-
-endif::[]
+include::../../libbeat/docs/shared-docker.asciidoc[]
 
+[float]
+[[monitoring-host]]
+=== Monitoring the Host Machine
 When executing Metricbeat in a container, there are some important
 things to be aware of if you want to monitor the host machine or other
 containers. Let's walk-through some examples using Docker as our container
 orchestration tool.
```

Comment:
A few clarifications or warnings could be provided for Docker on Windows + Mac users where the bind mounts won't work as expected.